validator.js

```diff
@@ -35,7 +36,8 @@ export const validateBom = (bomJson) => {
   );
   const ajv = new Ajv({
     schemas: [schema, defsSchema, spdxSchema],
-    strict: false,
+    strict: true,
```

Comment:
Let's keep `strict: false`, since there could be many other data-quality issues.